### ISSUE TITLE
Provide autowiring for DateTimeHelper

### ISSUE THREAD

**Issue by soullivaneuh** [sonata-project/SonataIntlBundle#212]
Ref:

```
  Cannot autowire service "AppBundle\Mailer\LegacyMailer": argument "$dateTimeHelper" of method "__construct()" references class "Sonata\IntlBundle\Templating\Helper\DateTimeHelper" but no such service exists. You should maybe alias this class to the existing "sonata.i  
  ntl.templating.helper.datetime" service.                                                                                                                                                                                                                                     
```

**Comment by XWB** [comment]
This should be fixed by https://github.com/sonata-project/SonataIntlBundle/pull/217

**Comment by OskarStark** [comment]
@Soullivaneuh can you confirm this one is fixed, and If so, close this issue please.

Thank you

**Comment by phansys** [comment]
The service `Sonata\IntlBundle\Templating\Helper\DateTimeHelper` is available as of version [2.5.0](https://github.com/sonata-project/SonataIntlBundle/releases/tag/2.5.0):

```
 // This service is a public alias for the service sonata.intl.templating.helper.datetime                               

Information for Service "sonata.intl.templating.helper.datetime"
================================================================

 DateHelper displays culture information. More information here http://userguide.icu-project.org/formatparse/datetime.

 ---------------- ---------------------------------------------------- 
  Option           Value                                               
 ---------------- ---------------------------------------------------- 
  Service ID       sonata.intl.templating.helper.datetime              
  Class            Sonata\IntlBundle\Templating\Helper\DateTimeHelper  
  Tags             templating.helper (alias: datetime)                 
  Public           yes                                                 
  Synthetic        no                                                  
  Lazy             no                                                  
  Shared           yes                                                 
  Abstract         no                                                  
  Autowired        no                                                  
  Autoconfigured   no                                                  
 ---------------- ---------------------------------------------------- 
```
Closing this issue.
